sdk/openai/openai/test/utils/utils.ts

```diff
@@ -99,6 +114,62 @@ export async function withDeployments<T>(
   );
 }
 
+export type DeploymentTestingParameters<T> = {
+  clientsAndDeployments: ClientsAndDeploymentsInfo;
+  run: (client: OpenAI, model: string) => Promise<T>;
+  validate?: (result: T) => void;
+  modelsListToSkip?: Partial<ModelInfo>[];
+  acceptableErrors?: AcceptableErrors;
+};
+
+/**
+ * Test with deployments invokes `test` call, so it should be inside of `describe` and not `it`.
+ * @param clientsAndDeployments -
+ * @param run -
+ * @param validate -
+ * @param modelsListToSkip -
+ * @param acceptableErrors -
```

Comment:
```suggestion
 * @param clientsAndDeployments - Information about the clients and their associated deployments.
 * @param run - A function that runs the test on a given client and model.
 * @param validate - An optional function to validate the result of the run function.
 * @param modelsListToSkip - An optional list of models to skip during the test.
 * @param acceptableErrors - An optional list of acceptable errors that should not cause the test to fail.
```

Reply:
Let's attempt to address this in a future PR.